packages/react/src/components/SpinButton/__snapshots__/SpinButton.test.tsx.snap

```diff
@@ -62,24 +62,27 @@ exports[`SpinButton snapshots renders correctly 1`] = `
     className=
 
         {
+          border-color: #605e5c;
           box-sizing: border-box;
           display: flex;
           height: 32px;
           min-width: 86px;
           position: relative;
         }
         &:after {
-          border-color: #605e5c;
           border-radius: 2px;
-          border-style: solid;
           border-width: 1px;
-          bottom: 0px;
+          border: 2px solid #605e5c;
+          bottom: -1px;
```

Comment:
The styles in this snapshot changed. `top`, `right`, `bottom `and `left `are now `-1px` rather than `2px`. `border-width` is now `2px` instead of `1px`.

The Screener snapshots are also different with the new border making the component visually larger. Based on the description for this PR this doesn't seem like an intentional change. Did you mean to make this visual change?

Reply:
yeah, i noticed that. i'll tweak and update

Reply:
fixed. the function returns inset vs top/bottom/left/right, but it should be the same, and consistent with other borders